runtime/onert/backend/cpu/ops/FullyConnectedLayer.cc

```diff
@@ -167,6 +168,12 @@ void FullyConnectedLayer::run()
 
 void FullyConnectedLayer::prepare()
 {
+  const int bias_size = getTensorShape(_bias).FlatSize();
```

Comment:
Hmm.. if _bias is nulltpr, it could cause segment fault.

Reply:
@YongseopKim Thanks! I've fixed this issue.